scss/_variables.scss

```diff
@@ -773,7 +773,11 @@ $dt-font-weight:              $font-weight-bold !default;
 $list-inline-padding:         $spacer * .25 !default;
 
 $mark-padding:                0 .1875em !default; // Boosted mod
-$mark-bg:                     $primary !default;
+$mark-color:                  $white !default; // Boosted mod
+$mark-bg:                     $black !default; // Boosted mod
+
+$mark-color-dark:             $black !default; // Boosted mod
```

Comment:
We got `$accordion-dark-*`, `$breadcrumb-dark-*`, etc. variables; it would mean here `$mark-dark-color` and `$mark-dark-bg` wouldn't it?

Reply:
Took the `*-bg-dark` Sass variables spelling. May I change all of these too ?